tree/ntuple/v7/src/RNTupleDescriptor.cxx

```diff
@@ -218,8 +218,16 @@ std::uint32_t DeserializeFrame(std::uint16_t protocolVersion, const void *buffer
    std::uint16_t protocolVersionMinRequired;
    bytes += DeserializeUInt16(bytes, &protocolVersionAtWrite);
    bytes += DeserializeUInt16(bytes, &protocolVersionMinRequired);
-   R__ASSERT(protocolVersionAtWrite >= protocolVersionMinRequired);
-   R__ASSERT(protocolVersion >= protocolVersionMinRequired);
+   if (protocolVersionAtWrite < protocolVersionMinRequired) {
```

Comment:
How can this happen since both `protocolVersionAtWrite` and `protocolVersionMinRequired` are retrieved from the source?

Reply:
Ouch, thanks for pointing it out!  The check is now removed entirely, as this version of the code simply supports version 0.  Once we support a newer forward, we can add a check for older versions.